node/service/src/grandpa_support.rs

```diff
@@ -241,7 +245,7 @@ pub(crate) struct PauseAfterBlockFor<N>(pub(crate) N, pub(crate) N);
 impl<Block, B> grandpa::VotingRule<Block, B> for PauseAfterBlockFor<NumberFor<Block>>
 where
 	Block: BlockT,
-	B: sp_blockchain::HeaderBackend<Block>,
+	B: sp_blockchain::HeaderBackend<Block> + 'static,
```

Comment:
```suggestion
	B: sp_blockchain::HeaderBackend<Block>,
```

Reply:
still required?

Reply:
I did the forceful variant: Remove all static and add as few as the compiler would be happy with, so this is obsolete.